custodia/server/__init__.py

```diff
@@ -100,7 +101,14 @@ def _create_plugin(parser, section, menu):
 
 
 def parse_config(args):
-    parser = ConfigParser(interpolation=ExtendedInterpolation())
+    defaults = {
+        'fqdn': socket.getfqdn(),
```

Comment:
shouldn't this be socket.getfqdn(socket.gethostname()) ?
The problem withg just getfqdn() if I read correctly is that it will do a reverse lookup address on the hosts ip address (which one ?) and sounds it will return unpredictable results ni many sistuations (multihomed, reverse not under admin control, etc...)
All that and I am a member of CARL (Campaign Against reverse Lookups) :-)